File: S3-5.py
data = sc.textFile("/readmespark.md")
linesWithSpark = textFile.filter(line => line.contains("Spark"))
linesWithSpark.cache()
linesWithSpark.count()

from org.apache.spark.storage import StorageLevel
puntos = [1,5,8,6,30]
rdd1 = sc.parallelize(puntos)
result = rdd1.map(x => x * x)
result.persist(StorageLevel.DISK_ONLY)
result.collect()

File: S3-5.scala

val data = sc.textFile("/readmespark.md")
val linesWithSpark = data.filter(line => line.contains("Spark"))
linesWithSpark.cache()
linesWithSpark.count()

import org.apache.spark.storage.StorageLevel
val puntos = Array(1,5,8,6,30)
val rdd1 = sc.parallelize(puntos)
val result = rdd1.map(x => x * x)
result.persist(StorageLevel.DISK_ONLY)
println(result.count())
println(result.collect().mkString(","))

